File: app/src/main/java/com/mlsdev/mapsappsample/listener/OnSeekBarListener.java
package com.mlsdev.mapsappsample.listener;

import android.widget.SeekBar;

import androidx.databinding.ObservableField;

public class OnSeekBarListener implements SeekBar.OnSeekBarChangeListener {
    private final ObservableField<String> progress;

    public OnSeekBarListener(ObservableField<String> progress) {
        this.progress = progress;
    }

    @Override
    public void onProgressChanged(SeekBar seekBar, int i, boolean b) {
        progress.set(String.valueOf(seekBar.getProgress()));
    }

    @Override
    public void onStartTrackingTouch(SeekBar seekBar) {
    }

    @Override
    public void onStopTrackingTouch(SeekBar seekBar) {
    }
}

File: app/src/main/java/com/mlsdev/mapsappsample/listener/OnSeekBarListener.kt
package com.mlsdev.mapsappsample.listener

import android.widget.SeekBar

import androidx.databinding.ObservableField

class OnSeekBarListener(private val progress: ObservableField<String>) : SeekBar.OnSeekBarChangeListener {

    override fun onProgressChanged(seekBar: SeekBar, i: Int, b: Boolean) {
        progress.set(seekBar.progress.toString())
    }

    override fun onStartTrackingTouch(seekBar: SeekBar) {}

    override fun onStopTrackingTouch(seekBar: SeekBar) {}
}
